modules/core/help.py

```
@@ -0,0 +1,295 @@
+        else:
+            help_name = module.split()[0]
+        if help_name in current_unloaded_modules:
+            await msg.finish(msg.locale.t("parser.module.unloaded", module=help_name))
+        elif help_name in err_modules:
+            await msg.finish(msg.locale.t("error.module.unloaded", module=help_name))
+        elif help_name in module_list:
+            module_ = module_list[help_name]
+            if module_.desc:
+                desc = module_.desc
+                if locale_str := re.match(r'\{(.*)}', desc):
+                    if locale_str:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)